src/MICore/LaunchOptions.cs

```diff
@@ -421,7 +421,7 @@ public sealed class UnixShellPortLaunchOptions : LaunchOptions
         /// <summary>
         /// Default location of the debugger on the remote machine.
         /// </summary>
-        public string DebuggerInstallationDirectory { get; private set; } = "~/.vs-debugger";
+        public string DebuggerInstallationDirectory { get; private set; } = ".vs-debugger";
```

Comment:
>" [](start = 76, length = 1)

One thing we loose with this change is that it is no longer possible to install to a non-home-relative directory using the AD7 metric. I wonder if we should check if the first character of this is a '/' and if so assume that the path is rooted.